src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyAssignment.cs

```diff
@@ -190,7 +171,7 @@ private JToken GetResource()
             var policyassignmentObject = new PolicyAssignment
             {
                 Name = this.Name,
-                Sku = this.Sku == null? new PolicySku { Name = "A0", Tier = "Free" } : this.Sku.ToDictionary(addValueLayer: false).ToJson().FromJson<PolicySku>(),
+                //Sku = this.Sku == null? new PolicySku { Name = "A0", Tier = "Free" } : this.Sku.ToDictionary(addValueLayer: false).ToJson().FromJson<PolicySku>(),
```

Comment:
Can remove this line altogether

Reply:
I'll handle Sku the same as in Set-AzureRmPolicyAssignment. Pending review with the Powershell team, we will either add the [Obsolete] attribute, which requires removing all references to Sku, or we will leave it supported, but change the logic to only send Sku to the backend if it was actually provided as input.